src/python/pants/engine/internals/build_files_test.py

```diff
@@ -98,76 +108,24 @@ def test_address_specs_duplicated() -> None:
     assert awo.origin == address_spec
 
 
-def test_address_specs_tag_filter() -> None:
```

Comment:
The below deleted tests were rewritten to use `self.request_single_product`. They were losing value in mocking everything out.